codebasin/walkers/platform_mapper.py

```diff
@@ -26,7 +26,7 @@ def _map_node(self, _node, _map):
         """
         # Do not map files that the user does not consider to be part of
         # the codebase
-        if isinstance(_node, FileNode) and _node.filename not in self.codebase["files"]:
+        if isinstance(_node, FileNode) and (_node.filename not in self.codebase["files"] or _node.filename in self.codebase["exclude_files"]):
```

Comment:
This line looks a little long to me.

What do you think about moving the second part of the conditional into a function (or a lambda)?  This syntax is probably wrong, but we could do something like:
```suggestion
        exclude = lambda f : f not in self.codebase["files"] or f in self.codebase["exclude_files"]
        if isinstance(_node, FileNode) and exclude(_node.filename):
```